lib/node/fullnode.js

```diff
@@ -94,6 +94,8 @@ class FullNode extends Node {
 
     // Pool needs access to the chain and mempool.
     this.pool = new Pool({
+      agent: this.config.str('agent'),
+      agentComment: this.config.str('agentComment'),
```

Comment:
Can we just add one?

Reply:
The PR was about the first. You guys asked for this extra.
Why both?

> Ofc someone really wants to, they could use agent to totally rewrite and that's okay as well, but in general this should give better alternative ?